.github/ISSUE_TEMPLATE/aws-access-request.yml

```diff
@@ -41,59 +41,65 @@ body:
+      description: Please provide the work email for the target principal
+      placeholder: jane.doe@company.com
     validations:
       required: true
   - type: input
     id: team-name
     attributes:
-      label: Your Team, Role, and Company
-      description: Please provide the name of your team, your role on that team, and the name of the company you work for
-      placeholder: e.g. Platform Security, Backend Engineer, Ad Hoc
+      label: Team, Role, and Company of the target principal
+      description: Please provide the name of the team, their role on that team, and the name of the company the target principal works for
+      placeholder: Team Moose, Antler Development, Alces LLC
     validations:
       required: true
   - type: input
     attributes:
       label: Product Manager (PM) name and email
-      description: Provide the name and email of your Product Manager
+      description: Provide the name and email of the Product Manager working with the target principal
     validations:
       required: true
   - type: input
     attributes:
       label: Product Owner (PO) name and email
-      description: Provide the name and email of your Product Owner
+      description: Provide the name and email of the Product Owner for the target principal
     validations:
       required: true
   - type: textarea
     id: aws-access
     attributes:
       label: Desired AWS Access
-      description: Please list which resources (bucket names, parameter store paths, EC2 instance name, etc.) would you like to access. Also list what type of access will you need (read, read-write, etc.). If there is a teammate whose permissions you'd like to copy, please list their name.
+      description: Please list which resources (bucket names, parameter store paths, EC2 instance name, etc.) and which type of access is requested. Explicitly list which type(s) of access are needed for which resources and keep these scopes as small/tight as possible. If there is a teammate whose permissions should be copied, please list their name.
       placeholder: I need access to S3 to read uploaded files from the /tools-team/uploads bucket.
   - type: input
     id: expiration-date
     attributes:
       label: Access Expiration
-      description: The requested access will be revoked at the start of this date. Enter a date (and time if applicable) or 'ongoing' for non-expiring access.
+      description: The requested access will be revoked at the start of this date. Enter a date (and time if applicable) or 'ongoing' for non-expiring access. If ongoing is selected, add a justification for this to the 'Additional Notes' field.
       placeholder: timestamp
     validations:
       required: true
+  - type: textarea
+    id: additional-notes
+    attributes:
+      label: Additional Notes
+      description: Use this section to add notes, such as justification for the access request, COR information when 'Other' was selected, etc.
```

Comment:
Change "COR" to "VOR"? Unless we want them to add additional info for both.

Reply:
Good catch, will be fixed via 3063b55bdebcfc27a81ab74478ff295b0a3ec2e3